test/automated/index.js

```diff
@@ -0,0 +1,211 @@
+/* eslint-env node, mocha, es6 */
+
+'use strict';
+
+const { before, after, beforeEach, describe, it } = require('mocha');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+require('chromedriver');
+const selenium = require('selenium-webdriver');
+const until = require('selenium-webdriver/lib/until');
+
+describe('Likely', function () {
+    let driver;
+
+    before(function () {
+        driver = new selenium.Builder()
+            .forBrowser('chrome')
+            .build();
+    });
+
+    after(function () {
+        return driver.quit();
+    });
+
+    describe('common operations', function () {
+        const mockedCounterValue = '10';
+
+        before(function () {
+            // The browser could start long
+            this.timeout(20000);
```

Comment:
It has, but this timeout is for Mocha, not for Selenium. It tells Mocha to stop the test only after 20 seconds instead of default 3. Mocha doesn’t know what’s running inside of the test and therefore can’t know if Selenium has already started or not.